samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/CMakeLists.txt

```diff
@@ -19,4 +20,5 @@ zephyr_include_directories(
 	src/
 	src/mesh
 	$ENV{ZEPHYR_BASE}/subsys/bluetooth/controller/hal
```

Comment:
I'm still wondering about this line. I realize it was already there before this PR, but could you clarify what it's needed for?

Reply:
I've not added it. (..to be safe I don't remember it or don't know how it get included).

I will remove it & I can compile it without this line.